package.json

```diff
@@ -5,7 +5,7 @@
     "email": "moebiusmania@gmail.com"
   },
   "engines": {
-    "node": "8.5.0"
+    "node": "^8.5.0"
```

Comment:
remove caret plz, fixed version number is better 😉

Reply:
without the caret it will run only on node 8.5.0, not on 8.5.1 or 8.9.1, isn't more safe to allow to use the last version of the major?